packages/bentley/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@bentley/grouping-mapping-widget",
+  "version": "0.0.15",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/bentley/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/main.bundle.js",
+  "types": "lib/grouping-mapping-widget.d.ts",
+  "scripts": {
+    "build": "webpack && npm run copy-files ",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "copy-files": "ncp README.md lib/README.md && ncp LICENSE.md lib/LICENSE.md",
+    "cover": "",
+    "lint": "npm run lint:ts",
+    "lint:fix": "npm run lint:ts -- --fix",
+    "lint:ts": "eslint -f visualstudio \"./src/**/*.{js,ts,tsx}\" --max-warnings=0",
+    "rebuild": "npm run build",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "test": "",
+    "test:watch": "npm -s test -- --reporter min --watch-extensions ts,tsx --watch"
+  },
+  "dependencies": {
+    "@bentley/bentleyjs-core": "^2.19.25",
+    "@bentley/geometry-core": "^2.19.25",
+    "@bentley/frontend-authorization-client": "^2.19.25",
+    "@bentley/imodeljs-common": "^2.19.25",
+    "@bentley/imodeljs-frontend": "^2.19.25",
+    "@bentley/itwin-client": "^2.19.25",
+    "@bentley/presentation-common": "^2.19.25",
+    "@bentley/presentation-components": "^2.19.25",
+    "@bentley/presentation-frontend": "^2.19.25",
+    "@bentley/ui-abstract": "^2.19.25",
+    "@bentley/ui-components": "^2.19.25",
+    "@bentley/ui-core": "^2.19.25",
+    "@bentley/ui-framework": "^2.19.25",
+    "@itwin/imodel-browser-react": "^0.12.1",
```

Comment:
This dependency and bunch of the others don't seem to be needed for the widget and a product of the Demo Portal.

`@itwin/imodel-browser-react`
`simple-react-validator`
`@itwin/itwinui-css`
`@itwin/itwinui-icons-react`
`@itwin/itwinui-react`
`cross-fetch`
`file-loader`